src/pod.ts

```diff
@@ -77,7 +78,7 @@ export class Pod {
       environmentOptions || {
         host: 'localhost',
         name: 'default',
-        scheme: 'http',
+        scheme: 'https',
```

Comment:
Yes, that's right. The issue I want to solve with this is I want to make it easy for environments (in Grow, "deployments") to customize the host without having to specify the scheme as well. I agree it's not great as-is though since https and localhost won't ever really work.

Maybe we should just have no default, and the value for `host` can be undefined? Something else?

Reply:
We could add to the `Environment` constructor an `if (environment === 'localhost')` and force it to be `https`. I do like the idea of having it default for https when it is not localhost.

Reply:
Good idea; updated.